src/formatters/diffs/const.js

```diff
@@ -0,0 +1 @@
+exports.STRING_OVERFLOW_MARKER = '...';
```

Comment:
Why we have this file, with this single const, with ES5 exports? BTW we don't use generic `const` or `utils` anymore